src/org/zaproxy/zap/extension/viewstate/HttpPanelViewStateView.java

```diff
@@ -0,0 +1,269 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Copyright 2010 psiinon@gmail.com
```

Comment:
Is this really old code, or should this be updated to 2016 and just ZAP Dev Team?